db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_03_22_235224) do
+ActiveRecord::Schema.define(version: 2020_04_02_014727) do
```

Comment:
@caseyli not sure why this got pulled in. Should I leave this or revert back before merging? This is essentially what will be merged with https://github.com/caseyli/localeatsottawa/pull/29